src/RouteAggregator.js

```diff
@@ -0,0 +1,132 @@
+import invariant from 'invariant';
+import Relay from 'react-relay';
+
+import getParamsForRoute from './getParamsForRoute';
+
+export default class RouteAggregator {
+  constructor() {
+    this.uniqueQueryNames = new WeakMap();
```

Comment:
They don't - the `runtime` optional in Babel makes these get silently polyfilled. The transpiled code actually has:

```js
var _WeakMap = require('babel-runtime/core-js/weak-map')['default'];
```